src/components/app.js

```diff
@@ -20,7 +19,13 @@ const App = createFactory(createClass({
       stackingContext
     } = this.props;
 
-    return StackingContextTree({tree: stackingContext.tree});
+    return div(
+        {}, // props
+        // App will only render SplitView, which will render the Tree on one side
+        //  and the DomContainer + select dropdown in another
+        StackingContextTree({tree: stackingContext.tree}),
+        DomContainer({html: "(DOM Container goes here)"})
```

Comment:
nit: Generally content is on the left and panels are on the right, so the order here would be reversed with DomContainer then StackingContextTree.